iceberg/iceberg-handler/src/test/results/positive/truncate_iceberg_table.q.out

```diff
@@ -225,7 +225,7 @@ Retention:          	0
 #### A masked pattern was here ####
 Table Type:         	EXTERNAL_TABLE      	 
 Table Parameters:	 	 
-	COLUMN_STATS_ACCURATE	{\"BASIC_STATS\":\"true\",\"COLUMN_STATS\":{\"id\":\"true\",\"value\":\"true\"}}
```

Comment:
`COLUMN_STATS` should be stale after the first truncation, otherwise we may get the wrong statistics, for example, max(id) is 10, which is not expected, as max(id) should be 5.